src/commands/cronjob/update.tsx

```diff
@@ -0,0 +1,153 @@
+import { ExecRenderBaseCommand } from "../../lib/basecommands/ExecRenderBaseCommand.js";
+import { Args, Flags } from "@oclif/core";
+import { ReactNode } from "react";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../rendering/process/process_flags.js";
+import { Success } from "../../rendering/react/components/Success.js";
+import assertSuccess from "../../lib/apiutil/assert_success.js";
+import { MittwaldAPIV2 } from "@mittwald/api-client";
+
+type UpdateResult = void;
+
+export default class Update extends ExecRenderBaseCommand<
```

Comment:
Also (noticed this only after reviewing): Does this PR also include the commits from #465? Maybe separate the branches into two separate ones?

Reply:
removed from this PR beacuse it doesnt exist here yet, I will take the other conversations belonging to this file over to the branch where they belong.